packages/core/src/main/cluster/manager.ts

```diff
@@ -246,7 +246,7 @@ export class ClusterManager {
 
   protected onNetworkOnline = () => {
     this.dependencies.logger.info(`${logPrefix} network is online`);
-    this.dependencies.store.clustersList.forEach((cluster) => {
+    this.dependencies.store.clustersList.get().forEach((cluster) => {
```

Comment:
_nit:_ why not? 🤪
anyway.. i don't think there is a big need to convert normal observable fields to computed values on the way _(too much)_.

```suggestion
    this.dependencies.get().store.get().clustersList.get().forEach((cluster) => {
```